pkg/monitoring/rules/rules.go

```diff
@@ -0,0 +1,208 @@
+package rules
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/konveyor/forklift-controller/pkg/monitoring/rules/recordingrules"
+)
+
+const (
+	forkliftPrometheusRuleName = "prometheus-forklift-rules"
+
+	prometheusLabelKey   = "prometheus.forklift.konveyor.io"
+	prometheusLabelValue = "true"
+
+	k8sAppLabelKey     = "app"
+	forkliftLabelValue = "forklift"
+)
+
+func SetupRules(namespace string) error {
+	err := recordingrules.Register(namespace)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func BuildPrometheusRule(namespace string, ownerRef *metav1.OwnerReference) (*promv1.PrometheusRule, error) {
+	rules, err := operatorrules.BuildPrometheusRule(
+		forkliftPrometheusRuleName,
+		namespace,
+		map[string]string{
+			prometheusLabelKey: prometheusLabelValue,
+			k8sAppLabelKey:     forkliftLabelValue,
+		},
+	)
+	rules.OwnerReferences = []metav1.OwnerReference{*ownerRef}
+	if err != nil {
+		return nil, err
+	}
+
+	return rules, nil
+}
+
+func PatchMonitorinLable(namespace string, clientset *kubernetes.Clientset) (err error) {
```

Comment:
MonitoringLabel?

Reply:
yes, in order to scrap the metrics to telemetry we need to have a specific label set to the pods we wish to expose, but now that I think about, maybe it will make more sense to update it to the deployment directly instead of patching it later on, checking this.

Reply:
Ok it seams we don't create the project, its automatically created, so we need to patch the labels later, looking in kubevirt repo they doing something similar.

Reply:
in case we must have it, there is a typo in the function name